src/openforms/utils/tests/feature_flags.py

```diff
@@ -0,0 +1,19 @@
+from copy import deepcopy
+
+from django.conf import settings
+from django.test import override_settings
+
+
+def enable_feature_flag(flag: str):
+    assert flag in settings.FLAGS, f"Flag {flag} is not defined in settings.FLAGS"
+    flags = {
+        **deepcopy(settings.FLAGS),
```

Comment:
Is there a specific reason a `deepcopy` is necessary?

Reply:
it's a better-safe-than-sorry approach, I really want to avoid the settings possibly being mutated accidentally which would break test isolation and cause hard-to-debug (flaky) test failures